src/parser.rs

```diff
@@ -1073,4 +1086,24 @@ a5: *x
             event.0 != Event::StreamEnd
         } {}
     }
+
+    #[test]
+    fn test_keep_tags_across_multiple_documents() {
+        let text = r#"
+%YAML 1.1
+%TAG !t! tag:test,2024:
+--- !t!1 &1
+foo: "bar"
+--- !t!2 &2
+baz: "qux"
+"#;
+        let mut loader = YamlLoader::default();
+        let mut parser = Parser::new(text.chars()).keep_tags(true);
+        assert!(parser.load(&mut loader, true).is_ok());
+        assert_eq!(loader.documents().len(), 2);
+        let yaml = &loader.documents()[0];
+        assert_eq!(yaml["foo"].as_str(), Some("bar"));
+        let yaml = &loader.documents()[1];
+        assert_eq!(yaml["baz"].as_str(), Some("qux"));
+    }
```

Comment:
```suggestion

        let mut loader = YamlLoader::default()
        let mut parser = Parser::new(text.chars()).keep_tags(false);
        assert!(parser.load(&mut loader, true).is_err());
    }
```

Can we add this just for good measure? I know it's also tested by the `yaml-test-suite`, but it's easier to see from the test what the `keep_tags` does.